Dockerfile

```diff
@@ -309,6 +309,16 @@ FROM scratch AS ipxe-generate
 COPY --from=pkg-ipxe-amd64 /usr/libexec/snp.efi /amd64/snp.efi
 COPY --from=pkg-ipxe-arm64 /usr/libexec/snp.efi /arm64/snp.efi
 
+FROM scratch AS microsoft-secureboot-database
+ADD https://github.com/microsoft/secureboot_objects.git /
```

Comment:
hmm so once cached we don't get any updates?

Reply:
yes, unless the upstream repository changes, and we'll see a change here in `git diff` (but it shouldn't change of course)